zerver/webhooks/gocd/tests.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import urllib
+from typing import Text
+
+from zerver.lib.test_classes import WebhookTestCase
+from zerver.models import get_realm, get_user
+
+class GocdHookTests(WebhookTestCase):
+    STREAM_NAME = 'gocd'
+    URL_TEMPLATE = "/api/v1/external/gocd?stream={stream}&api_key={api_key}&topic=builds"
+    FIXTURE_DIR_NAME = 'gocd'
+    TOPIC = 'builds'
+
+    def test_gocd_message(self) -> None:
+        expected_message = (u"Author: Balaji B <balaji@example.com>\n"
+                            u"Build status: Passed :thumbsup:\n"
+                            u"Details: URL: https://github.com/gocd/gocd, Branch: master\n"
```

Comment:
I think you could just take this line out and turn it into the topic. So the topic would be `gocd/gocd - master` which would link to `https://github.com/gocd/gocd` :)